apps/emqx/src/emqx_router.erl

```diff
@@ -103,8 +103,7 @@
 -type batch_route() :: {emqx_types:topic(), dest()}.
 
 -record(routeidx, {
-    entry :: '$1' | emqx_topic_index:key(dest()),
-    unused = [] :: nil()
```

Comment:
this is intended.

Reply:
this record is also a mnesia schema, having an extra field is to make future upgrades easier.